Modules/Bridge/NumPy/wrapping/test/itkMeshToPolyDataFilterTest2.py

```diff
@@ -0,0 +1,47 @@
+#==========================================================================
+#
+#   Copyright NumFOCUS
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#          http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+#==========================================================================*/
+
+import itk
+
+Dimension = 3
+PixelType = itk.ctype('double')
+MeshType = itk.Mesh[PixelType, Dimension]
+mesh = MeshType.New()
+PointType = itk.Point[itk.F, Dimension]
+point0 = PointType()
+point0[0] = -1
+point0[1] = -1
+mesh.SetPoint(0, point0)
+point1 = PointType()
+point1[0] = 1
+point1[1] = -1
+mesh.SetPoint(1, point1)
+point2 = PointType()
+point2[0] = 1
+point2[1] = 1
+mesh.SetPoint(2, point2)
+point3 = PointType()
+point3[0] = 1
+point3[1] = 1
+mesh.SetPoint(3, point3)
+
+filt = itk.MeshToPolyDataFilter.New(mesh)
```

Comment:
We can focus on the `PyVectorContainer` wrapping in this PR and leave out `MeshtoPolyData`.

Reply:
I will remove that file.